src/annotator/util/frame-util.js

```diff
@@ -43,6 +43,16 @@ function isValid (iframe) {
   return iframe.className !== 'h-sidebar-iframe';
 }
 
+function isReady (iframe, callback) {
```

Comment:
Good point. Let's clarify the naming between the two.

**isReady** 
- When `document.readyState` is not `loading`, meaning it's either `interactive` or `complete`
- MDN definition of `interactive`: _The document has finished loading and the document has been parsed but sub-resources such as images, stylesheets and frames are still loading._
- I need a callback when the frame is at minimum ready

**isLoaded**
- When `document.readyState` is `complete`
- MDN Definition of `complete`: _The document and all sub-resources have finished loading. The state indicates that the load event is about to fire._
- I need a callback when frame is fully loaded and about to go into an idle state